medusa/tv/series.py

```diff
@@ -588,33 +588,33 @@ def get_all_episodes(self, season=None, has_location=False):
         :return:
         :rtype: list of Episode
         """
-        sql_selection = b'SELECT season, episode, '
-
         # subselection to detect multi-episodes early, share_location > 0
-        sql_selection += (b'(SELECT '
-                          b'  COUNT (*) '
-                          b'FROM '
-                          b'  tv_episodes '
-                          b'WHERE '
-                          b'  showid = tve.showid '
-                          b'  AND season = tve.season '
-                          b"  AND location != '' "
-                          b'  AND location = tve.location '
-                          b'  AND episode != tve.episode) AS share_location ')
-
-        sql_selection = sql_selection + b' FROM tv_episodes tve WHERE showid = ' + str(self.series_id)
+        sql_selection = (b'SELECT season, episode, (SELECT '
+                         b'  COUNT (*) '
+                         b'FROM '
+                         b'  tv_episodes '
+                         b'WHERE '
+                         b'  showid = tve.showid '
+                         b'  AND season = tve.season '
+                         b"  AND location != '' "
+                         b'  AND location = tve.location '
+                         b'  AND episode != tve.episode) AS share_location '
+                         b'FROM tv_episodes tve WHERE showid = ?'
```

Comment:
Hmm where missing the indexer here. I missed that one.

Reply:
I think you should also pick a different inner join then showid = tve.showid. Maybe use `episode_id`